src/public/modules/forms/admin/views/collaborator.client.modal.html

```diff
@@ -10,7 +10,8 @@
   >
     <div ng-if="isDisplayTransferOwnerModal">
       <div class="label-custom" id="invite-title">
-        Transfer ownership to {{transferOwnerEmail}}
+        Transfer ownership to
+        <span id="transfer-owner-email">{{transferOwnerEmail}}</span>
```

Comment:
prefer not using ids and stick to classes. ids can only be used once on the entire layout (and whilst the odds of `transfer-owner-email` being used somewhere else it's still plausible haha

ids should be kept to maybe a container div/etc, not for a span.